### PR TITLE
Wip/removeadminservice

### PR DESCRIPTION
**Pull request by olivergs**

### CODE REVIEW

File: admin/fleetcommander/admin.py [hunk 1]
```diff
@@ -627,20 +627,37 @@ def websocket_stop(self):
...
+        help='Listening host for admin application')
+    parser.add_argument(
+        '--port', action='store', metavar='HOST', type=int, default=None,
+        help='Listening port for admin application')
+    parser.add_argument(
+        '--datadir', action='store', metavar='DATADIR', default=None,
+        help='Directory for static admin data')
+    parser.add_argument(
+        '--statedir', action='store', metavar='STATEDIR', default=None,
+        help='Directory for data storage')
```
Comment (aruiz) [inline admin/fleetcommander/admin.py]: We should check if --host was passed and if not exit with a failure and an error message stating that the host must be specified explicitely

File: admin/fleetcommander/admin.py [hunk 2]
```diff
@@ -627,20 +627,37 @@ def websocket_stop(self):
...
+    parser.add_argument(
+        '--host', action='store', metavar='HOST', default=None,
+        help='Listening host for admin application')
+    parser.add_argument(
+        '--port', action='store', metavar='HOST', type=int, default=None,
+        help='Listening port for admin application')
+    parser.add_argument(
+        '--datadir', action='store', metavar='DATADIR', default=None,
+        help='Directory for static admin data')
+    parser.add_argument(
```
Comment (aruiz) [inline admin/fleetcommander/admin.py]: We should be more specific here, --profiledata and --appdata are probably better option names here. "static admin data" is not very descriptive for a casual user.

File: admin/fleetcommander/utils.py [hunk 3]
```diff
@@ -71,16 +82,22 @@ def config_to_dict(config):
         return res
 
     config = config_to_dict(config)
-    args['host'] = config[SECTION_NAME].get('host', args['host'])
-    args['port'] = config[SECTION_NAME].get('port', args['port'])
+    if host is None:
```
Comment (aruiz) [inline admin/fleetcommander/utils.py]: What's the point of doing all this if we can put these values in the 'default' field of the argparser?

File: data/fleet-commander-standalone.in [hunk 4]
```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+function usage {
```
Comment (aruiz) [inline data/fleet-commander-standalone.in]: Why don't we show exactly the same options that the python module accepts and just pass $@/$* accordingly?

ArgumentParser also does the usage text for you automatically, we should delegate this on the python module

File: admin/fleetcommander/utils.py (hunk 3 above)
Comment (olivergs) [inline admin/fleetcommander/utils.py]: That's for handle both test and standalone execution cases.

**Pull request closed without merging**